### ISSUE TITLE
Create policy parameter for share of pension contributions subtracted from JSA applicable income

### ISSUE THREAD

**Issue by MaxGhenis** [PSLmodels/openfisca-uk#140]
This is currently hard-coded to 0.5:

https://github.com/PSLmodels/openfisca-uk/blob/7426a3b3434a8d2204d19af40e21df49729bebda/openfisca_uk/variables/finance/benefit/family/JSA_income.py#L111-L114

**Comment by MaxGhenis** [comment]
This also shows up in housing benefit:

https://github.com/PSLmodels/openfisca-uk/blob/7426a3b3434a8d2204d19af40e21df49729bebda/openfisca_uk/variables/finance/benefit/family/housing_benefit.py#L129-L132

Perhaps the deductible pension contributions could be its own variable, based on the policy parameter.

**Comment by nikhilwoodruff** [comment]
Yep I agree. There's also a *lot* of places where variable formulas could do with a bit more abstraction (applicable income for benefits, many other places).
